Aliases path errors

### Current Behavior
When I ran Haul on windows, in 91% it crashed and wrote errors: 


First three errors:

```
ERROR in ./~/haul/src/utils/polyfillEnvironment.js
Module not found: Error: Can't resolve 'InitializeCore' in 'C:\Users\Admin\Documents\Git-hub\snootik-app\node_modules\haul\src\utils'
 @ ./~/haul/src/utils/polyfillEnvironment.js 35:0-25
 @ multi ./~/haul/src/utils/polyfillEnvironment.js ./index.android.js

ERROR in ./~/react-native/Libraries/react-native/react-native-implementation.js
Module not found: Error: Can't resolve 'AccessibilityInfo' in 'C:\Users\Admin\Documents\Git-hub\snootik-app\node_modules\react-native\Libraries\react-native'
 @ ./~/react-native/Libraries/react-native/react-native-implementation.js 19:31-59
 @ ./src/index.js
 @ ./index.android.js
 @ multi ./~/haul/src/utils/polyfillEnvironment.js ./index.android.js

ERROR in ./~/react-native/Libraries/react-native/react-native-implementation.js
Module not found: Error: Can't resolve 'ActivityIndicator' in 'C:\Users\Admin\Documents\Git-hub\snootik-app\node_modules\react-native\Libraries\react-native'
 @ ./~/react-native/Libraries/react-native/react-native-implementation.js 20:31-59
 @ ./src/index.js
 @ ./index.android.js
 @ multi ./~/haul/src/utils/polyfillEnvironment.js ./index.android.js
```

### Haul Configuration (webpack.haul.js)
```
const path = require('path');

module.exports = ({ platform, root }, defaults) => ({
  entry: `./index.${platform}.js`,
  resolve: {
    ...defaults.resolve,
    alias: {
      ...defaults.resolve.alias,
      theme: path.join(root, 'src', 'styles'),
      ui: path.join(root, 'src', 'ui'),
      helpers: path.join(root, 'src', 'helpers'),
      config: path.join(root, 'src', 'config'),
    },
  },
});

```

### Your Environment
Windows 10
Visual Studio Code
Android emulator

| software         | version
| ---------------- | -------
| Haul      |  1.0.0-beta.0   
| react-native   |   0.45.0  
| node      |  6.10.0       
| npm or yarn     | 0.21.3

--- a/src/utils/resolveModule.js
+++ b/src/utils/resolveModule.js
@@ -6,13 +6,15 @@
  */
 
 const resolver = require('resolve');
+const path = require('path');
 
 /**
 * Resolves the path to a given module
 * We point to 'package.json', then remove it to receive a path to the directory itself
 */
 
-module.exports = (root: string, name: string) =>
-  resolver
-    .sync(`${name}/package.json`, { basedir: root })
-    .replace(/(\\|\\\\|\/)package.json$/, '');
+module.exports = (root: string, name: string) => {
+  const filePath = resolver.sync(`${name}/package.json`, { basedir: root });
+
+  return path.dirname(filePath);
+};

--- a/src/utils/resolveModule.js
+++ b/src/utils/resolveModule.js
@@ -15,4 +15,4 @@ const resolver = require('resolve');
 module.exports = (root: string, name: string) =>
   resolver
     .sync(`${name}/package.json`, { basedir: root })
-    .replace('/package.json', '');
+    .replace(/(\\|\\\\|\/)package.json$/, '');

